decentraland/unity-renderer
# Feat: UI Gradient Chat

## What does this PR change?



This PR adds the animated linear gradient to all the chat windows that requires it. The images that apply the gradient have a script to control it's properties.

## How to test the changes?



1. Go to: https://play.decentraland.zone/?renderer-branch=feat/UI-Gradient-Chat
2. Open the chat
3. Verify the gradient is rendering properly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
